ui/ducks/send/send.js

```diff
@@ -1875,9 +1883,11 @@ export function updateRecipientUserInput(userInput) {
 
     const inputIsValidHexAddress = isValidHexAddress(userInput);
     let isProbablyAnAssetContract = false;
+    let smartContractAddress;
```

Comment:
since this is not longer passed to the validation method this can go back to being a const inside the `if (inputIsValidHexAddress) {` conditional

Reply:
done

Reply:
linting should tell me this stuff